src/Symfony/Component/Form/FormTypeInterface.php

```diff
@@ -87,6 +87,11 @@ public function setDefaultOptions(OptionsResolverInterface $resolver);
      * for returning type instances may be dropped from future releases.
      *
      * @return string|null|FormTypeInterface The name of the parent type if any, null otherwise.
+     *
+     * @deprecated Returning a {@link FormTypeInterface} instance is deprecated
```

Comment:
please don't use the ``@deprecated`` tag when deprecating only a given signature and not the whole method, because it does not respect the meaning of the phpdoc and so confusing tools building on the normal phpdoc behavior.

Just put the explanation in the phpdoc description